data/migrations/20200218211829_home-run-data.js

```diff
@@ -0,0 +1,251 @@
+const crypto = require('crypto')
+
+exports.up = function (knex) {
+	return knex.schema
+		.createTable('members', col => {
+			col.increments()
+			col.string('username', 40).notNullable()
+			col.string('email', 128).notNullable()
+			col.string('password').notNullable()
+			col.string('token')
+			col.string('current_household')
```

Comment:
Isnt current household suppose to be a foreign key pointing to the household table?

Reply:
I believe you're correct. @vinnihoke